File: Búsqueda/Secuencial/secuencial.rb
def busqueda_secuencial(arr, objetivo)
    arr.each_with_index do |elemento, indice|
        return indice if elemento == objetivo
    end
    nil
end

# Ejemplo de uso
arreglo = [5, 3, 8, 4, 2]
objetivo = 4
resultado = busqueda_secuencial(arreglo, objetivo)

if resultado
    puts "Elemento encontrado en el índice #{resultado}"
else
    puts "Elemento no encontrado"
end

File: Búsqueda/Secuencial/secuencial.js
function busquedaSecuencial(arr, objetivo) {
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === objetivo) {
            return i; // Devuelve el índice donde se encuentra el objetivo
        }
    }
    return -1; // Devuelve -1 si el objetivo no se encuentra en el arreglo
}

// Ejemplo de uso
const arreglo = [5, 3, 8, 4, 2];
const objetivo = 4;
const resultado = busquedaSecuencial(arreglo, objetivo);

if (resultado !== -1) {
    console.log(`Elemento encontrado en el índice ${resultado}`);
} else {
    console.log('Elemento no encontrado');
}